components/eam/src/physics/crm/samxx/vars.cpp

```diff
@@ -1835,6 +1869,8 @@ real2d crm_output_t_vt_tend;
 real2d crm_output_q_vt_tend;
 real2d crm_output_t_vt_ls;
 real2d crm_output_q_vt_ls;
+real2d crm_output_u_vt_tend;
```

Comment:
just a minor cleanup would be nice here. You can group *_vt_tend together and group *_vt_ls together.

Reply:
bah, I thought I got all these! Thanks for catching it.

Reply:
fixed